src/ServiceControl.Transports.Msmq/transport.manifest

```diff
@@ -5,7 +5,7 @@
       "Name": "MSMQ",
       "DisplayName": "MSMQ",
       "TypeName": "ServiceControl.Transports.Msmq.MsmqTransportCustomization, ServiceControl.Transports.Msmq",
-      "SampleConnectionString": "",
+      "SampleConnectionString": null,
```

Comment:
Not sure if this part was a good idea. Looks like that made a handful of tests fail. Before adjusting the tests I would make sure this doesn’t affect what happens to the validation when setting up an instance in SCMU.

Reply:
The UI in SCMU [is doing the same check](https://github.com/Particular/ServiceControl/blob/master/src/ServiceControl.Config/UI/InstanceAdd/ServiceControlEditorViewModel.cs#L47) i.e. `IsNullOrEmpty`. That said the tests seem to assume that `string.Empty` is the expected value so I'm 👍 for the change.